alf/utils/dist_utils.py

```diff
@@ -69,27 +69,27 @@ def _call(self, x):
 
     def _inverse(self, y):
         """
-        y > 0:
-            y = x / (1+x) => x = y / (1 - y)
-        y < 0:
-            y = x / (1-x) => x = y / (1 + y)
+        :math:`y > 0`:
+            :math:`y = \frac{x}{1+x}\Rightarrow x = \frac{y}{1 - y}`
+        :math:`y < 0`:
+            :math:`y = \frac{x}{1-x}\Rightarrow x = \frac{y}{1 + y}`
```

Comment:
This format will be broken. You can put them in one math block 

`.. math::
     ...
`

Reply:
updated